skopt/tests/test_parameter.py

```diff
@@ -16,57 +18,74 @@ def check_distribution(Dist, vals, random_val):
-        assert_equal(p, expected[i])
 
+def test_real():
+    a = Real(1, 25)
+    for i in range(10):
+        yield (check_distribution_limits, a.rvs(random_state=i), 1, 25)
+    random_values = a.rvs(random_state=0, n_samples=10)
+    assert_array_equal(random_values.shape, (10))
 
-def check_simple_grid(values, expected_rvs, dist_type):
-    grid = check_grid([values])
 
-    dist = grid[0][0]
-    rvs = dist.rvs(n_samples=2, random_state=1)
-    assert_true(isinstance(dist, dist_type))
-    assert_almost_equal(rvs, expected_rvs, decimal=3)
+# def test_categorical_transform():
+#     categories = ['apple', 'orange', 'banana']
+#     cat = Categorical(*categories)
 
+#     # LabelEncoder sorts classes alphabetically -> banana == 1
+#     assert_array_equal(cat.transform(['apple', 'banana']), [0, 1])
+#     assert_array_equal(cat.inverse_transform([0, 1]), ['apple', 'banana'])
 
-def test_check_grid():
-    yield (check_simple_grid, (1, 4), [2, 1], Integer)
-    yield (check_simple_grid, (1., 4.), [2.668, 3.881], Real)
-    yield (check_simple_grid, (1, 2, 3), [2, 3], Categorical)
 
+# def test_real_transform():
+#     dist = Real(0, 10)
+#     values = [9.1, 2.3]
 
-def test_sub_grids():
-    expected = [('a', 1), ('a', 2), ('b', 5), ('b', 5), ('a', 1)]
+#     assert_array_equal(dist.transform(values), values)
+#     assert_array_equal(dist.inverse_transform(values), values)
 
-    for i,p in enumerate(sample_points([(['a'], (1, 4)), (['b'], (4, 6))],
-                                       len(expected), random_state=3)):
-        assert_equal(p, expected[i])
 
+# def test_simple_grid():
+#     expected = [(2, 1), (1, 2), (2, 2), (2, 1), (1, 2)]
 
-def test_categorical_transform():
-    categories = ['apple', 'orange', 'banana']
-    cat = Categorical(*categories)
+#     for i, p in enumerate(sample_points([(1, 3), (1, 4)],
+#                                        len(expected), random_state=1)):
+#         assert_equal(p, expected[i])
 
-    # LabelEncoder sorts classes alphabetically -> banana == 1
-    assert_array_equal(cat.transform(['apple', 'banana']), [0, 1])
-    assert_array_equal(cat.inverse_transform([0, 1]), ['apple', 'banana'])
+
+# def check_simple_grid(values, expected_rvs, dist_type):
+#     grid = check_grid([values])
+
+#     dist = grid[0][0]
+#     rvs = dist.rvs(n_samples=2, random_state=1)
+#     assert_true(isinstance(dist, dist_type))
+#     assert_almost_equal(rvs, expected_rvs, decimal=3)
+
+
+# def test_check_grid():
+#     yield (check_simple_grid, (1, 4), [2, 1], Integer)
+#     yield (check_simple_grid, (1., 4.), [2.668, 3.881], Real)
+#     yield (check_simple_grid, (1, 2, 3), [2, 3], Categorical)
+
+
+# def test_sub_grids():
+#     expected = [('a', 1), ('a', 2), ('b', 5), ('b', 5), ('a', 1)]
+
+#     for i,p in enumerate(sample_points([(['a'], (1, 4)), (['b'], (4, 6))],
+#                                        len(expected), random_state=3)):
+#         assert_equal(p, expected[i])
```

Comment:
Why removing these tests? Is it simply because it is still a work in progress? We need tests for `check_grid`, as it will further be the entry point to validate and check user parameters.